src/components/Project/Features/index.js

```diff
@@ -160,7 +160,7 @@ export default class Features extends React.Component {
                 Make sure all relevant features are pushed into {project.gitBranch}. 
                 {project.gitProtocol === "SSH" &&
                   <div>
-                  If you haven't done so already, <NavLink to={"/projects/" + project.slug + "/settings"}><strong> make sure your deploy key is added in your git settings.</strong></NavLink>
+                  If you haven't done so already, <NavLink to={"/projects/" + project.slug + "/" + this.props.store.app.currentEnvironment.name + "/settings"}><strong> make sure your deploy key is added in your git settings.</strong></NavLink>
```

Comment:
This would be cool ````/projects/${project.slug}/${this.props.store.app.currentEnvironment.name}/settings````
but I'm not block you over it.

Reply:
You have to use backticks though instead of double-quotes.

Reply:
Indeed, I have switched it to using string interpolation.